install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -32,35 +32,37 @@ jaeger:
   hub: docker.io/jaegertracing
   image: all-in-one
   tag: 1.12
-  memory:
-    max_traces: 50000
-  # spanStorageType value can be "memory" and "badger" for all-in-one image
-  spanStorageType: badger
   persist: false
-  storageClassName: ""
-  accessMode: ReadWriteMany
+  env:
+    SPAN_STORAGE_TYPE: "badger"
+    BADGER_EPHEMERAL: "false"
+    BADGER_DIRECTORY_VALUE: "/badger/data"
+    BADGER_DIRECTORY_KEY: "/badger/key"
+    COLLECTOR_ZIPKIN_HTTP_PORT: "9411"
+    MEMORY_MAX_TRACES: 50000
```

Comment:
This `MEMORY_MAX_TRACES` option could probably be left out, so options only relate to the storage type that has been selected. If a user wants memory storage, then they would need to define other options (and probably would remove the badger related ones).